scripts/tests/test_env_mutation_gate.py

```
@@ -0,0 +1,179 @@
+import subprocess
+import sys
+from pathlib import Path
+
+import pytest
+
+SCRIPTS = Path(__file__).resolve().parents[1]
+GATE = SCRIPTS / "check_env_mutation.py"
+SHIM = SCRIPTS / "check-env-mutation.sh"
+
+_SPEC = importlib.util.spec_from_file_location("check_env_mutation", GATE)
+assert _SPEC is not None and _SPEC.loader is not None
```

Ruff (0.16.2)

[warning] Assertion should be broken down into multiple parts

Break down assertion into multiple parts

(PT018)